### ISSUE TITLE
十三，一些会过期的函数。。只是warning

### ISSUE THREAD

**Issue by zdpau** [zdpau/-TensorFlow-#13]
一、**softmax_cross_entropy_with_logits** (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.

**Instructions for updating:** Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

以后使用这个？See @**{tf.nn.softmax_cross_entropy_with_logits_v2}**.

二， **read_data_sets** (from tensorflow.contrib.learn.python.learn.datasets.mnist) is deprecated and will be removed in a future version.
**Instructions for updating:** Please use alternatives such as official/mnist/dataset.py from tensorflow/models.

三，**maybe_download** (from tensorflow.contrib.learn.python.learn.datasets.base) is deprecated and will be removed in a future version.
**Instructions for updating:** Please write your own downloading logic.

四， **_internal_retry.<locals>.wrap.<locals>.wrapped_fn** (from tensorflow.contrib.learn.python.learn.datasets.base) is deprecated and will be removed in a future version.
**Instructions for updating:**Please use urllib or similar directly.

五，**extract_images** (from tensorflow.contrib.learn.python.learn.datasets.mnist) is deprecated and will be removed in a future version.
**Instructions for updating:** Please use tf.data to implement this functionality.

六，**extract_labels** (from tensorflow.contrib.learn.python.learn.datasets.mnist) is deprecated and will be removed in a future version.
**Instructions for updating:**Please use tf.data to implement this functionality.

七，**dense_to_one_hot** (from tensorflow.contrib.learn.python.learn.datasets.mnist) is deprecated and will be removed in a future version.
**Instructions for updating:** Please use tf.one_hot on tensors.

八，**DataSet.__init__** (from tensorflow.contrib.learn.python.learn.datasets.mnist) is deprecated and will be removed in a future version.
**Instructions for updating:** Please use alternatives such as official/mnist/dataset.py from tensorflow/models.
